### DISCUSSION TITLE
Full Responsive Layout

### DISCUSSION THREAD

**Discussion by henryavila** [laravel/nova-issues#3247] [Ideas & Feature Requests]
I have using Laravel Nova for a while, and I really love it.
The only problem I have with Nova is that the Layout is not responsive. I know that there's package for it, but it don't work 100%.

Laravel Nova is not used by developers, it's used by the end user to manager the site content.
Actually, most of the access is done via Mobile  (as a global trend) and how Laravel nova dont suport a responsive layout the end user experience is really bad.

About the problem with the columns in a mobile screen, a goot approach is what is made with DataTable, it automatically hide the colomns and add a button to display their data. Even in a small screen, the dataTables works perfect.
see: https://datatables.net/extensions/responsive/examples/column-control/columnPriority.html

So please, consider implement it in the new Laravel Nova Serie, it will greatly help a lot of Nova customers

**Comment by davidhemphill** [comment]
No details yet, but I can say that we're exploring the best way to implement this for the next series. 👍

**Reply by henryavila** [reply]
Amazing.
Thanks a lot!

**Reply by henryavila** [reply]
@davidhemphill , the https://backpackforlaravel.com uses the same concept as DataTable. It display the columns based on importance. On smaller device, hide columns with less importante. Will be amazing see this feature in Laravel Nova!
